### ISSUE TITLE
License should take and SPDX identifier/expression (or in an alternative field)

### ISSUE THREAD

**Issue by puerco** [ossf/security-insights-spec#86]
When parsing machine readable data, it is much more useful to have an [SPDX license identifier](https://spdx.org/licenses/) (or an [SPDX expression](https://spdx.github.io/spdx-spec/v3.0/annexes/SPDX-license-expressions/) . 

The current `license` field points to a URL, which is useful, but tools parsing the insights yaml can provide better results if the license type is already in a machine readable format.  This is the standard used by both SBOM formats.

**Comment by eddie-knight** [comment]
I've drafted this in both the project release & repository (source) sections in #96 

```yaml
  license:
    url: https://foo.bar/LICENSE
    expresion: MIT
```

**Comment by eddie-knight** [comment]
Thanks for this suggestion @puerco! 

The new [license](https://github.com/ossf/security-insights-spec/blob/main/specification-details/aliases.md#license) object type will be included in the upcoming release (#97)
